common/src/View/QtUtils.cpp

```diff
@@ -181,10 +181,6 @@ namespace TrenchBroom {
         QWidget* makeInfo(QWidget* widget) {
             makeDefault(widget);
 
-            auto font = widget->font();
```

Comment:
Please use `widget->setAttribute(Qt::WA_MacSmallSize);` instead.

Reply:
Sorted- widget rendering appears unchanged with the new attribute on Windows, so this seems good.